### ISSUE TITLE
Map is blank when root path is visited

### ISSUE THREAD

**Issue by straleyb** [osulp/building-oregon#124]
We might want to figure out something to do so that when people first visit our site, they aren't presented with a blank map.

**Comment by straleyb** [comment]
Relates #132

**Comment by terrellt** [comment]
@straleyb asked if the front page can show only buildings that are close to them. This'll require a few things:

1. Spatial index fields in Oregon Digital (https://github.com/OregonDigital/oregondigital/pull/834)
2. The ability for javascript to ask for points around a certain location (to populate the map after the user has given permission to view its location)
3. The ability to refresh the map with new points.

**Comment by straleyb** [comment]
This will technically be fixed after #139 is merged. Ill close this and open a new ticket specifically for showing buildings close to a person who's location is enabled.
